notebooks/mping_initial_xai.ipynb

```diff
@@ -0,0 +1,5548 @@
+{
```

Comment:
<p>General comment on the variable name format, you can use python's string replace function or the pandas.Series.str.replace to replace every "_" with " ". It can make the y axis more readable automatically. Alternatively you can create a fancy name version of the same list with some mix of automatic and manual string processing by removing underscores and replacing computational variable names with something more reader-friendly, like "TEMP_C" -&gt; "Temperature (C)" or "UGRD_m/s" -&gt; "U-Wind (m/s)". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2es/ptype-physical/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>